### PR TITLE
adding instructions for ctcdecode

### PR DESCRIPTION
**Pull request by sourabh2k15**

### CODE REVIEW

**Comment by fsschneider** [comment]
Hey Sourabh,
thanks for this PR.
Could you make sure that you sign the CLA first? Otherwise, we cannot merge it.
Let me know if you signed it and it is an issue with the bot, which happens from time to time. In that case, I will take care of it.

File: README.md [hunk 1]
```diff
@@ -123,6 +123,15 @@ Docker is the easiest way to enable PyTorch/JAX GPU support on Linux since only
 
    </details>
 
+## Packages to be installed from source
+
+some of the dependencies need to be installed from source like `ctcdecode`, here's the steps for source installation of `ctcdecode`
+
+1) git clone https://github.com/parlance/ctcdecode.git
+2) cd CTCDecoder
```
Comment (znado) [inline README.md:131]: nit: this should be `cd ctcdecode`

**Review by znado — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -123,6 +123,15 @@ Docker is the easiest way to enable PyTorch/JAX GPU support on Linux since only
 
    </details>
 
+## Packages to be installed from source
+
+some of the dependencies need to be installed from source like `ctcdecode`, here's the steps for source installation of `ctcdecode`
+
+1) git clone https://github.com/parlance/ctcdecode.git
```
Comment (znado) [inline README.md:130]: also this needs to be `git clone --recursive https://github.com/parlance/ctcdecode.git` to get it to work correctly

**Comment by sourabh2k15** [comment]
Hi Frank, I verified with Peter that mlcommons CLA has been signed for all googlers ... can you check if bot has some bug ? 
I had signed an internal CLA and was also able to submit code into init2winit and optax which are both open sourced repos of google

**Comment by morphine00** [comment]
Hi @sourabh2k15 , sorry, the right link for the CLA form is https://forms.gle/Ew1KkBVpyeJDuRw67 , please fill that out. Even though all Googlers are in fact "in" MLCommons, we still need to keep track of individual developers' usernames.

We know the CLA bot has a bad link, but tweaking that is proving a little annoying. If you folks know of anyone in MLCommons who's familiar with GitHub actions, please send them my way!

**Comment by znado** [comment]
As per discussion with sourabh2k15, closing as obsolete.

**Pull request closed without merging**